MongoDB.md

```diff
@@ -32,53 +32,72 @@ make mongo_install
 Configuration
 -------------
 
-* `WALG_STREAM_CREATE_COMMAND`
+* ##### `WALG_STREAM_CREATE_COMMAND`
 
 Command to create MongoDB backup, should return backup as single stream to STDOUT. Required for backup procedure.
 
-* `WALG_STREAM_RESTORE_COMMAND`
+* ##### `WALG_STREAM_RESTORE_COMMAND`
 
 Command to unpack MongoDB backup, should take backup (created by `WALG_STREAM_CREATE_COMMAND`) 
 to STDIN and push it to MongoDB instance. Required for restore procedure.
 
-* `MONGODB_URI`
+* ##### `MONGODB_URI`
 
 URI used to connect to a MongoDB instance. Required for backup and oplog archiving procedure.
 
-* `OPLOG_ARCHIVE_AFTER_SIZE`
+* ##### `OPLOG_ARCHIVE_AFTER_SIZE`
 
 Oplog archive batch in bytes which triggers upload to storage.
 
-* `OPLOG_ARCHIVE_TIMEOUT_INTERVAL`
+* ##### `OPLOG_ARCHIVE_TIMEOUT_INTERVAL`
 
-Timeout interval (passed since previous upload) to trigger upload to storage.
+Time interval (passed since previous upload) to trigger upload to storage.
 
-* `OPLOG_PUSH_STATS_ENABLED`
+Format: [golang duration string](https://golang.org/pkg/time/#ParseDuration).
+
+* ##### `MONGODB_LAST_WRITE_UPDATE_INTERVAL`
+
+Interval to update the latest majority optime. wal-g archives only majority committed operations.
+Format: [golang duration string](https://golang.org/pkg/time/#ParseDuration).
+
+
+* ##### `OPLOG_PUSH_WAIT_FOR_BECOME_PRIMARY`
+
+Wait for primary and start archiving or exit immediately. 
+
+* ##### `OPLOG_PITR_DISCOVERY_INTERVAL`
```

Comment:
Perhaps, a more in-depth description of this option may be necessary.
May be I miss something, but I did not get the meaning of this key.